packages/dashmate/configs/schema/configJsonSchema.js

```diff
@@ -511,6 +511,19 @@ module.exports = {
                   required: ['port'],
                   additionalProperties: false,
                 },
+                pprof: {
```

Comment:
`pprof` needs to be added also in the parent required list

`required: ['docker', 'p2p', 'rpc', 'consensus', 'node', 'moniker', 'genesis'],`
^^^